docs/navigation.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="apple-touch-icon" sizes="180x180" href="../assets/images/favicons/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="../assets/images/favicons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="../assets/images/favicons/favicon-16x16.png">
+    <link rel="manifest" href="../assets/images/favicons/site.webmanifest">
+
+    <link rel="stylesheet" href="./styles/index.css" />
+    <link rel="stylesheet" href="./../components/navigation.css">
+
+    <title>Unstoppable&nbsp;UI | Navigation</title>
```

Comment:
Any reason to use &nbsp; and not just give a space?

Reply:
No particular reason, rn. But initially I thought to give multiple spaces but later on changed to single space. And is it present everywhere didn't change. I'll change this during site revamp. It would probably not change anything or add complexity.